### PR TITLE
add api functions

### PR DESCRIPTION
**Pull request by guillermooo**
API to rst — first cut.

### CODE REVIEW

**Comment by guillermooo** [comment]
We'll need to do manual post-editing not matter what, but let's try to get the cleanest starting point possible.

**Comment by guillermooo** [comment]
BTW, API docs in RtD look horrible.

File: source/reference/api_2.rst [hunk 1]
```diff
@@ -0,0 +1,1074 @@
...
+
+.. function:: arch()
+
+   Returns the CPU architecture, which may be "x32" or "x64"
+
+   :rtype: String
+
+.. class:: sublime.View
+
+   .. method:: Methods
```
Comment (FichteFoll) [inline source/reference/api_2.rst]: It seems you got the header rows as well.

**Comment by FichteFoll** [comment]
I suppose we can close and delete this in favor of #124?

**Comment by FichteFoll** [comment]
@guillermooo ping

**Comment by guillermooo** [comment]
Close away!

**Pull request closed without merging**